frontend/src/app/data-entry/episode-form/episode-form.component.ts

```diff
@@ -1,15 +1,19 @@
 import { Component } from "@angular/core";
 import { ActivatedRoute } from "@angular/router";
 import { actionIcons, dataIcons } from "@shared/icons";
-import { DataEntryEpisodeFormGQL } from "generated/graphql";
+import { DataEntryEpisodeFormGQL, Entity } from "generated/graphql";
 import { filter, map, share, switchMap } from "rxjs";
+import { FormService } from "../shared/form.service";
 
 @Component({
     selector: "lc-episode",
     templateUrl: "./episode-form.component.html",
     styleUrls: ["./episode-form.component.scss"],
+    providers: [FormService],
```

Comment:
In an upcoming PR I move this provider line from the individual components to the module. We don't need multiple instantiations of FormService, and it breaks the singleton structure.

Reply:
Conceptually, the service is used so the component can aggregate the status of its child components. If you were to create multiple instances of `EpisodeFormComponent` (or any of the other forms that provide a FormService), you would absolutely need multiple instances of the service, too.

Angular services are often stateless or keep track of a global state in the application, which makes them work best as singletons, but a singleton structure is not inherently superior; in this case, the service is tracking the local state of the component, so making it a singleton sounds like it invites bugs in my opinion.

Reply:
I did not intend to suggest that the singleton pattern is superior, but I see your point. I'll adjust my PR.